Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  File.swift
+//
+//
+//  Created by Keith  on 2022/6/21.
+//
+
+extension CommandParser {
+  func canInteract(error: Error, split: inout SplitArguments) -> Bool {
+    if let error = error as? ParserError {
+      switch error {
+      case let .missingValueForOption(inputOrigin, name):
+        var input: String?
+        while input?.isEmpty ?? true {
+          print("Please enter value for '\(name.synopsisString)': ", terminator: "")
+          input = readLine() ?? nil
+        }
+        
+        let index = inputOrigin.elements.first!.baseIndex! + 1
+        split._elements.append(.init(value: .value(input!),
+                                     index: .init(inputIndex: .init(rawValue: index))))
```

Comment:
Does appending the new value work if the option with the missing value is in the middle? For example:

```
repeat hello --count --include-counter
```